.github/workflows/PROJECT-AUTO-CHANGELOG-CONTROL.yaml

```
@@ -287,17 +287,55 @@ jobs:
           GH_TOKEN: ${{ secrets._GITHUB_PAT_TOKEN }}
         run: |
           PR_HEAD=$(gh pr view ${{ github.event.pull_request.number }} --json headRefName -q .headRefName)
+          PR_BASE=$(gh pr view ${{ github.event.pull_request.number }} --json baseRefName -q .baseRefName)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)